hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerMetrics.java

```diff
@@ -111,10 +111,10 @@ public void incContainerOpsMetrics(ContainerProtos.Type type) {
   }
 
   public void incContainerOpsLatencies(ContainerProtos.Type type,
-                                       long latencyMillis) {
```

Comment:
i don't understand why checkstyle doesn't complain. The indentation looks wrong.
But that's fine. I am +1

Reply:
because Indentation checkstyle forceStrictCondition defaults to false

Reply:
```diff
diff --git a/hadoop-hdds/dev-support/checkstyle/checkstyle.xml b/hadoop-hdds/dev-support/checkstyle/checkstyle.xml
index 3a69c793c26..487136de543 100644
--- a/hadoop-hdds/dev-support/checkstyle/checkstyle.xml
+++ b/hadoop-hdds/dev-support/checkstyle/checkstyle.xml
@@ -204,6 +204,7 @@
         <module name="Indentation">
             <property name="basicOffset" value="2" />
             <property name="caseIndent" value="0" />
+            <property name="forceStrictCondition" value="true"/>
         </module>
         <!--<module name="TodoComment"/>-->
         <module name="UpperEll"/>
```

Reply:
fixed.